### PR TITLE
Feat/ add navigation bar style 🔥🥊

### PR DESCRIPTION
**Pull request by oradwanomar**
### **Add a reusable navigation bar to the whole project.**

-  **can hide any UIbarbutton one or two** 
-  **change icons from basic icon -> icon with red badge with recall the navBar with Badged image**  ❤️👌🏻 

Screen Shot 2022-07-22 at 8 34 21 AM

Screen Shot 2022-07-22 at 8 34 30 AM

### CODE REVIEW

**Review by HebaMad — APPROVED** [review]

File: UIDokan/Sources/UIDokan/ReusableViews/NavigationBar.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+//  
+//
+//  Created by Omar Ahmed on 21/07/2022.
+//
+
+import UIKit
+
+public extension UIViewController{
+    
+    func configureNavigationBar(headTitle: String,
```
Comment (KarimEbrahemAbdelaziz) [inline UIDokan/Sources/UIDokan/ReusableViews/NavigationBar.swift:12]: I would recommend 2 things here.
1. Rename `firstButtonName` to something else that reflect on the usage, as you use it as an Asset name which is image name.
2. Look for better asset management library (or build your own ;)) to manage your assets in more convenient way (ex: R.swift pod). 
What do you think?

Comment (ahmdyasser) [reply UIDokan/Sources/UIDokan/ReusableViews/NavigationBar.swift:12]: @KarimEbrahemAbdelaziz  كيمو بنفسه 🫡

Comment (KarimEbrahemAbdelaziz) [reply UIDokan/Sources/UIDokan/ReusableViews/NavigationBar.swift:12]: @ahmdyasser 7abeby teslm 😄☺️

Comment (oradwanomar) [reply UIDokan/Sources/UIDokan/ReusableViews/NavigationBar.swift:12]: > I would recommend 2 things here.
> 
> 1. Rename `firstButtonName` to something else that reflect on the usage, as you use it as an Asset name which is image name.
> 2. Look for better asset management library (or build your own ;)) to manage your assets in more convenient way (ex: R.swift pod).
>    What do you think?

 I will take these recommendations into account and implement them. Thanks, @KarimEbrahemAbdelaziz for the advice ❤️✌🏻

**Pull request closed without merging**